react/week2/todolistsapp/src/components/TodoList.jsx

```diff
@@ -0,0 +1,35 @@
+import { todos } from "./todosCollection";
+import { useState } from "react";
+import TodoListItem from "./TodoListItem";
+
+export default function TodoList() {
+    const [todoList, setTodoList] = useState(todos);
+    const randomTodos = ["Do HYF homework", "Eat Lunch", "Cook Dinner", "Do Laundry", "Clean Dishes", "Go to danish class"]
+
+    function addTodo() {
+        setTodoList(todoList.concat({
+            id: todoList.length + 1,
```

Comment:
I'm not sure if you meant to have an unique id, which I guess you should. This way will give you repetitive Ids, by just adding and removing and adding again. a better solution to generate a random unique ID can be this : `Math.random().toString(16).slice(2)`